proto/server.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+package proto;
+
+// NotaryServer defines the TUF operations that the server supports
+service Notary {
+	rpc Auth(Token) returns (AuthResponse) {}
+	rpc Update(Updates) returns (UpdateResponse) {}
+	rpc Get(RepoRole) returns (Repository) {}
+	rpc GetKey(RepoRole) returns (KeyResponse) {}
```

Comment:
Some keys (currently just timestamp, but we're working on snapshot as optional) are held by the server as it would be a pain for users to have to manage timestamping themselves. This function is a way for a client to request a timestamp key to sign into the root.json during initialization.

We're also talking about how to let users do server held key rotation and this function will probably also be used for that.